python/modules/Stop0lBaselineProducer.py

```diff
@@ -64,6 +64,8 @@ def beginFile(self, inputFile, outputFile, inputTree, wrappedOutputTree):
         self.out.branch("Pass_HEMVeto30"     + self.suffix, "O", title="HEM Veto 2018: eta[-3, -1.4], phi[-1.57, -0.87], pt > 30")
         self.out.branch("Pass_exHEMVeto20"   + self.suffix, "O", title="HEM Veto 2018: eta[-3.2, -1.2], phi[-1.77, -0.67], pt > 20")
         self.out.branch("Pass_exHEMVeto30"   + self.suffix, "O", title="HEM Veto 2018: eta[-3.2, -1.2], phi[-1.77, -0.67], pt > 30")
+	self.out.branch("Jet_nsortedIdx"     + self.suffix, "I")
+	self.out.branch("Jet_sortedIdx"      + self.suffix, "I", lenVar="Jet_nsortedIdx")
```

Comment:
You don't need Jet_nsortedIdx right? The len of Jet_sortedIdx should be nJets as stored in the ntuple.